src/web/app/components/notification-banner/notification-banner.component.ts

```diff
@@ -31,6 +34,12 @@ export class NotificationBannerComponent implements OnInit {
     }
   }
 
+  ngOnChanges(): void {
+    if (this.url.includes('notifications')) {
+      this.closeNotification();
+    }
```

Comment:
Could add a comment here e.g. `// Hide the notification if the user is on the user notifications page`